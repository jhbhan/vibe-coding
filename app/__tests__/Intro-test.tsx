import { render } from '@testing-library/react-native';
import HomeScreen from '../(tabs)';

test('renders correctly', () => {
    const tree = render(<HomeScreen />);
    expect(tree).toMatchSnapshot();
});