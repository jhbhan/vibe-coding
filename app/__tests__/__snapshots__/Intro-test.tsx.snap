// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#3b5998",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    collapsable={false}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#3b5998",
        "height": 1334,
        "justifyContent": "center",
        "left": 0,
        "opacity": 1,
        "position": "absolute",
        "top": 0,
        "width": 750,
        "zIndex": 2,
      }
    }
  >
    <Text
      style={
        {
          "color": "white",
          "fontSize": 48,
          "fontWeight": "bold",
          "letterSpacing": 2,
        }
      }
    >
      Compare
    </Text>
  </View>
</View>
`;
